spring-cloud-dataflow-docs/src/main/asciidoc/index.adoc

```diff
@@ -22,6 +22,7 @@ include::preface.adoc[]
 include::spring-cloud-dataflow-overview.adoc[]
 include::architecture.adoc[]
 include::getting-started.adoc[]
+include::configuration.adoc[]
```

Comment:
I think this file will need to be included in other projects (eg CF, YARN, k8s, etc) as well